Dockerfile

```diff
@@ -1,8 +1,9 @@
-ARG base_version=1.2.0
-ARG registry=quay.io
-
-FROM ${registry}/ncigdc/python38-builder:${base_version} as build
+ARG BASE_VERSION=3.0.1-rc3
```

Comment:
Feels painful to have to keep changing every repo for Docker file changes and gitlab changes. If feels like we could do something cross-cutting.